src/util.ts

```diff
@@ -50,6 +50,15 @@ export const applicationProperties = {
       templateName: 'PresentationWithContent.pptx'
     },
     sideloadingDirectory: path.join(os.homedir(), 'Library/Containers/com.microsoft.Powerpoint/Data/Documents/wef')
+  },
+  outlook:{
+    documentationLink:"https://docs.microsoft.com/en-us/outlook/add-ins/sideload-outlook-add-ins-for-testing"
```

Comment:
One thing I noticed, the documentationLink was added for add-in hosts that do not support sideload, but we might as well add the documentationLink for the other add-in hosts for completeness.